src/quo2/components/drawers/drawer_top/view.cljs

```diff
@@ -0,0 +1,157 @@
+                                          (str description " ∙ " (i18n/label :t/on-device))]
+    (= :context-tag type)                [rn/view
+                                          {:accessibility-label :context-tag-wrapper
+                                           :style               {:flex-wrap :wrap}}
+                                          [context-tag/view
+                                           {:type           :community
+                                            :community-name community-name
+                                            :community-logo community-logo
+                                            :size           24}]]
+    (and (not= :label type) description) [text/text
+                                          {:size   :paragraph-1
+                                           :weight :regular
+                                           :style  (style/description theme blur?)}
+                                          description]))
+
+(defn- right-icon
+  [{:keys [theme type on-button-press on-button-long-press button-disabled? button-icon]}]
+  (cond
+    (= :info type)                          [icons/icon
+                                             :i/info
+                                             {:accessibility-label :info-icon
+                                              :size                20
+                                              :color               (colors/theme-colors colors/neutral-50
+                                                                                        colors/neutral-40
+                                                                                        theme)}]
+    (and (= :context-tag type) button-icon) [button/button
+                                             {:accessibility-label :button-icon
+                                              :on-press            on-button-press
+                                              :on-long-press       on-button-long-press
+                                              :disabled?           button-disabled?
+                                              :type                :primary
+                                              :size                24
+                                              :icon-only?          true}
+                                             button-icon]))
+
+(defn- left-title
+  [{:keys [type label title theme blur?]}]
+  (case type
+    :label [text/text
+            {:weight :medium
+             :size   :paragraph-2
+             :style  (style/description theme blur?)}
+            label]
+    [text/text
+     {:size   :heading-2
+      :weight :semi-bold}
+     title]))
+
+(defn- view-internal
+  [{:keys [title type theme description blur? community-name community-logo button-icon on-button-press
+           on-button-long-press
+           button-disabled? account-avatar-emoji account-avatar-customization-color icon-avatar
```

Comment:
imo it is better to use `customization-color` prop here as that's all this is. There won't be separate customization-colors on different components. This is also consistent with our other quo2 components api which is nice to have in the design system components 👍

Reply:
Done